### ISSUE TITLE
EmojiIndex :An exception is thrown

### ISSUE THREAD

**Issue by yoonasy** [serebrov/emoji-mart-vue#89]
![image](https://user-images.githubusercontent.com/24489180/93973759-cd982780-fd85-11ea-8113-362fdae459e0.png)

### import
```
import 'emoji-mart-vue-fast/css/emoji-mart.css'
import data from 'emoji-mart-vue-fast/data/apple.json'
import { Picker, EmojiIndex } from 'emoji-mart-vue-fast'
```

### use
```
      i18n: {
        categories: {
          search: '搜索结果',
          recent: '经常使用',
          smileys: '心情',
          people: '人物',
          nature: '动物 & 大自然',
          foods: '食物 & 饮料',
          activity: '活动',
          places: '旅行 & 地标',
          objects: '物体',
          symbols: '符号',
          flags: '国旗',
          custom: '自定义',
        }
      }
...
    const emojiIndex = new EmojiIndex(data, {
      i18n: this.i18n,
      exclude: ['flags', 'symbols', 'objects', 'activity', 'foods', 'people', 'places']
    })
```

I exclude `places` in this example, I get an error

```
vue.runtime.esm.js?2b0e:1888 Error: Can not find emoji by id: department_store
    at e.value (emoji-mart.js?41fb:1)
    at VueComponent.idleEmoji (emoji-mart.js?41fb:1)
    at Watcher.get (vue.runtime.esm.js?2b0e:4479)
    at Watcher.evaluate (vue.runtime.esm.js?2b0e:4584)
    at Proxy.computedGetter (vue.runtime.esm.js?2b0e:4836)
    at Proxy.eval (emoji-mart.js?41fb:1)
    at VueComponent.Vue._render (vue.runtime.esm.js?2b0e:3548)
    at VueComponent.updateComponent (vue.runtime.esm.js?2b0e:4066)
    at Watcher.get (vue.runtime.esm.js?2b0e:4479)
    at new Watcher (vue.runtime.esm.js?2b0e:4468)
```

**Comment by yoonasy** [comment]
`emoji picker` It's not show，It's a `<!-- -->`

**Comment by serebrov** [comment]
@yoonasy the problem seems to be that the `department_store` is the default emoji used in the preview area and when the 'places' set is excluded, that emoji becomes unavailable.
I'll see if that can be handled in a better way, but to fix the issue you can just set the `emoji` property for the picker:

```
      <Picker emoji="smile" :data="emojiIndex" />
```

**Comment by serebrov** [comment]
@yoonasy I just published version 7.0.5 with a fix - now the component should not fail with an error and use the first emoji avaialbe for preview area. Probably, you would still need to configure the `emoji` property for the picker component, but, at least, you will see the component rendered.

Also a note regarding your example above: the `i18n` property should be set for the picker component, not for the `EmojiData` intance:

```
<template>
  <StaticPicker emoji="department_store" :data="indexI18n" :i18n="i18n" />
</template>

<script>
import data from '../data/all.json'
import { StaticPicker, EmojiIndex } from '../src'

const i18n = {
  categories: {
    search: '搜索结果',
    recent: '经常使用',
    smileys: '心情',
    people: '人物',
    nature: '动物 & 大自然',
    foods: '食物 & 饮料',
    activity: '活动',
    places: '旅行 & 地标',
    objects: '物体',
    symbols: '符号',
    flags: '国旗',
    custom: '自定义',
  },
}

const indexI18n = new EmojiIndex(data, {
  exclude: [
    'places',
  ],
})

export default {
  data() {
    return {
      indexI18n: indexI18n,
      i18n: i18n,
    }
  },
  components: {
    StaticPicker,
  },
}
</script>
```
